transport/src/main/java/io/netty/channel/group/ChannelGroup.java

```diff
@@ -133,6 +133,22 @@
     ChannelGroupFuture write(Object message, ChannelMatcher matcher);
 
     /**
+     * Writes the specified {@code message} to all {@link Channel}s in this
+     * group that match the given {@link ChannelMatcher}. If the specified {@code message} is an instance of
```

Comment:
that match -> that is matched by

.. because it's not a channel that matches a matcher?